ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -450,7 +517,7 @@ enum TransactionParser {
           for: transaction.fromAccountId.address,
           accounts: accountInfos
         ),
-        fromAddress: transaction.fromAccountId.address,
+        fromAccountId: transaction.fromAccountId,
```

Comment:
For each transcation there is associated "BraveWallet.AccountInfo", so let's store it instead of "fromAccountId"